interested-parties.md

```diff
@@ -15,6 +15,7 @@ Cerence | Hatice Ozguven
 Cloud Native Architects & Qwinix | Leonardo Murillo
 CNCF | Katie Gamanji
 Codefresh | Dan Garfield
+comforte AG | Piotr Zaniewski
```

Comment:
Thanks @Piotr1215 — A tip similar to https://github.com/gitops-working-group/gitops-working-group/pull/170#pullrequestreview-780057521
> Hi! Welcome! Could you add your GitHub username next to your name - we had not done this with previous interested parties and then realized we really had no efficient way to be able to reach out to all of you so I want to make sure we start fixing that ;)

Reply:
No worries added next to my name.